caas/Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM public.ecr.aws/ubuntu/ubuntu:20.04
 ARG TARGETOS
 ARG TARGETARCH
-ARG BUILDOS
```

Comment:
Not sure why this is deleted....

`WARN[0000] missing "BUILDOS" build argument. Try adding "--build-arg BUILDOS=<VALUE>" to the command line `

Reply:
It wasn't used AFAIK. Not sure why you are getting that warning, considering I removed it because of that warning.

Reply:
Ah ignore me, I think I messed up the merge into my branch.